app/assets/javascripts/spree/backend/solidus_quick_rma.js

```diff
@@ -1,2 +1,9 @@
 // Placeholder manifest file.
-// the installer will append this file to the app vendored assets here: vendor/assets/javascripts/spree/backend/all.js'
\ No newline at end of file
+// the installer will append this file to the app vendored assets here: vendor/assets/javascripts/spree/backend/all.js'
+//= require_tree ./templates
+//= require_tree ./return_authorizations
+
+$(function() {
```

Comment:
does this mean the backbone render is executed on every page load in the admin, even if it is not visible ?